etc/kayobe/kolla/globals.yml

```diff
@@ -77,6 +77,13 @@ prometheus_node_exporter_extra_volumes:
   - "textfile:/var/lib/node_exporter/textfile_collector"
 prometheus_node_exporter_cmdline_extras: "--collector.textfile.directory=/var/lib/node_exporter/textfile_collector"
 
+# Set Ceph manager exporter endpoints
+{% if groups['mgrs'] | length > 0 %}
+prometheus_ceph_mgr_exporter_endpoints:
+{% for host in groups['mgrs'] %}
+  - "{{ admin_oc_net_name | net_ip(host) }}:9283"
```

Comment:
Have you considered making the port number a variable? I don't know enough about it to actually know if it's a sensible idea.

Reply:
Its the standard port for the ceph dashboard. Also I dont think its exists as a variable within KA (unlike something like `prometheus_node_exporter_port` for example).

Reply:
This port is the default, and I don't think it is ever changed anywhere. The variable to change this port in KA for example was removed a while ago: https://github.com/openstack/kolla-ansible/commit/4e6fe7a6da4d5471cd0425c781f25f3e9567d3a0